### PR TITLE
Rails-cook Book version 1

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
**Hello Reviewer 👋 , thank you for taking the time to review our PR.**

### What does this PR do?

- Add Login page and registration page built with Devise.
### Food list:
- display a list of food added by the logged-in user as in the wireframe 
-  lead to a form that allows users to add new food.
### Recipes list:
-  display a list of recipes created by the logged-in user as in the wireframe.
-  lead to recipe details.
- If the user is the owner of the recipe, it allows the user to delete it.
### Public recipe list:
- display a list of all public recipes ordered by newest as in the wireframe.
- lead to recipe details.
### Recipe details:**
- If the recipe is public or the user is the owner of the recipe, it displays the recipe details as in the wireframe.
- If the user is the owner of the recipe, it will lead to the form that allows the user to add new food.
###  General shopping list view :
 - show the list of food that is missing for all recipes of the logged-in user 
- count the total food items and total price of the missing food.
### We made sure there are no N+1 queries happening.
- Create a navigation menu that allows users to open all of the pages you created.

**We worked in a group of two:**

@Benmuiruri  and @EzekielUtshudi 

### Thanks for your time and effort to review our code ,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
+- Twitter: [Optimize](https://twitter.com/_optimize)
+- LinkedIn: [Ben Kiarie](https://www.linkedin.com/in/benjamin-kiarie-180b66149/)
+
+
+# Author 2
+
+👤 **Ezekiel Utshudi**
+- GitHub: [@EzekielUtshudi](https://github.com/EzekielUtshudi)
+- Twitter: [@UtshudiEzekiel](https://twitter.com/UtshudiEzekiel)
+- LinkedIn: [@ezekiel-utshudi](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
```
Comment (VICTOR-CHUKWU) [inline README.md:70]: - [ ] There is no live link for your project, also there is no link to the video description of your project, Kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi Team, 
## STATUS: CHANGES REQUIRED :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Benmuiruri) [reply README.md:70]: Hi @VICTOR-CHUKWU I think you have this mistaken for a capstone project.

There was no mention of a video or deployment.

Please check the requirements here https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md

**Review by Lordkaito — APPROVED** [review]
Hi @Benmuiruri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**